hydra-main/src/main/java/com/addthis/hydra/job/spawn/balancer/SpawnBalancer.java

```diff
@@ -1205,14 +1246,54 @@ protected void updateAggregateStatistics(Iterable<HostState> hosts) {
                 findActiveJobIDs();
                 double maxMeanActive = -1;
                 double maxDiskPercentUsed = -1;
+                long maxDiskFree = 0;
+                long minDiskFree = Long.MAX_VALUE;
+                BigInteger sumDiskFree = BigInteger.valueOf(0);
```

Comment:
In our specific case, Long.MAX_VALUE = 2^63-1, and 1 petabyte = 2^50, so `long` should be more than enough here. It makes the code a bit easier to read, that's all.